hub/settings.py

```diff
@@ -185,6 +185,8 @@
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
+EMAIL_SECRET_KEY = os.environ.get("EMAIL_SECRET_KEY")
+EMAIL_INVITATION_BASE_URL = "http://localhost:3000"
```

Comment:
check if prod.. if prod it should be `https://hub.indiaultimate.org`

Reply:
hi @SibiAkkash @Joe2k ,

What setting configures dev or prod environment. Is it `DEBUG = TRUE`?